keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -351,27 +273,228 @@ Scalability:
     without a consistency guarantee), we expect scalability to be roughly the
     same as baseline (but with the benefit of improved correctness)
 
-### Rollout Plan
+### Graduation Criteria
 
-#### Serving consistent reads from cache
+#### Alpha
 
-Guard feature with the `WatchCacheConsistentReads` feature gate.
+- Feature is implemented behind a feature gate
+- Unpaginated LIST requests is served from watch cache
+- First page of paginated requests is served from watch cache
 
-#### Reflectors
+#### Beta
```

Comment:
There are 3 unresolved issues:

1) being able to force list from etcd
2) performance validation
3) fallback mechanism

I think all of these are fine to be resolved for beta only, as long as we claim in the KEP what we do for Alpha for (3).
But please mention them explicitly as Beta blockers already.

Reply:
What parts of performance validations should be done for Alpha which for Beta. I think it would be useful to have some validation for Alpha.